Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,104 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        public Equation(string operand1,string operand2,string operation,string result){
+            this.operand1=operand1;
+            this.operand2=operand2;
+            this.operation=operation;
+            this.result=result;
+        }
+        public enum EquationParts{
+            operand1,operand2,result
+        }
+        private string operand1;
+        private string operand2;
+        private string operation;
+        private string result;
+        public string Operand1 { get; private set; }
+        public string Operand2 { get; private set; }
+        public string Operation { get; private set; }
+        public string Result { get; private set; }
+        private bool IsValidValue(string value,string expectedValue){
+            if(expectedValue.Length!=value.Length)
+            return false;
+            return true;
+        }
+        private EquationParts GetEquationPartToBeCorrected(){
+            if(operand1.Contains('?'))
+            return EquationParts.operand1;
+            else if(operand2.Contains('?'))
+            return EquationParts.operand2;
+            else
+            return EquationParts.result;
+        }
+        private bool IsValidEquation(string operand1, string operand2, string result){
+            if(Convert.ToInt32(result)==Convert.ToInt32(operand1)*Convert.ToInt32(operand2))
+            return true;
+            return false;
+        }
+        private int MatchAndReturnFixedDigit(string actual, string expected){
+            int res=-1;

```

Comment:
This can just be like this 
 var index = actual.IndexOf('?');
 return int.Parse(expected[index].ToString());

Use provided core functions, rather than doing string manipulataions